portfolio/src/main/webapp/index.html

```diff
@@ -1,18 +1,27 @@
 <!DOCTYPE html>
 <html>
-  <head>
-    <meta charset="UTF-8">
-    <title>My Portfolio</title>
-    <link rel="stylesheet" href="style.css">
-    <script src="script.js"></script>
-  </head>
-  <body>
-    <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
-    </div>
-  </body>
-</html>
+
+<head>
+	<meta charset="UTF-8">
+	<title>Azfaar's Portfolio</title>
+	<link rel="stylesheet" href="style.css">
+	<script src="script.js"></script>
+</head>
+
+<body>
+	<div id="content">
+		<img src="/images/azfaar.jfif" alt="Azfaar's picture" width="200" height="200">
+		<h2>Hi, my name is</h2>
+		<h1>Azfaar Qureshi</h1>
+		<h3>Welcome to my portfolio.</h3></br>
+		<p>I'm currently a 3rd year Computer Engineering student at the University of Waterloo.
+			I have experience with Frontend, Backend and Production engineering.
+			I'm graduating in April 2022 and am currently searching for my next internship.
+			Take a look at my resume and previous experiences below!</p>
+		<p>Click here to get a random greeting:</p>
+		<button onclick="addRandomGreeting()">Hello</button>
+		<div id="greeting-container"></div>
+	</div>
```

Comment:
It looks like you're using tabs to indent code in this file. There's nothing wrong with tabs except I think everyone else on the team is using spaces and you'll run into all sorts of annoying problems when you start working on the group project in a couple of weeks.

For now, I'd recommend searching for "<my editor> use spaces instead of tabs" and switch to using spaces. In particular, I'd recommend using 2 spaces per tab (which is probably setting you can configure). Happy to chat more or help with making the change.

Reply:
done!